Sources/NostrSDK/Events/Channels/MuteChannelUserEvent.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MuteChannelUserEvent.swift
+//
+//
+//  Created by Konstantin Yurchenko, Jr on 9/11/24.
+//
+
+import Foundation
+
+/// User no longer wants to see messages from another user.
+/// See [NIP-28](https://github.com/nostr-protocol/nips/blob/master/28.md#kind-44-mute-user).
+public class MuteChannelUserEvent: NostrEvent {
```

Comment:
Same here about `reason`.

Reply:
Yes, implementing this feature soon too.